docs/guide/cleanup.md

```diff
@@ -31,102 +31,92 @@ We're mostly looking for packages which fit the following criteria:
+Back to our Vite example - at the time of writing this, we can see in the Rollup bundle that both `fast-glob` and `glob` are bundled into Vite.
 
-This will mean we are bundling two non-trivial packages which essentially do the same thing. We have found a possible performance improvement!
+This means we are bundling two non-trivial packages which essentially do the same thing. We have found a possible performance improvement!
 
 #### Create cleanup issues
 
-At this point, if you have identified some potential performance improvements, you should open issues in this repository.
-
-Even if you will try tackle them yourself, this will help improve visibility so people do not duplicate work and can help you out.
+At this point, if you have identified some potential performance improvements, you should open issues in the [ecosystem-cleanup](https://github.com/es-tooling/ecosystem-cleanup/issues) repository.
 
-Or if you do not have time to tackle it yourself, it is possible someone else can follow your findings.
+Even if you will try tackle them yourself, this will help improve visibility so people do not duplicate work and can help you out. Or, if you do not have time to tackle it yourself, it is possible for someone else to follow your findings.
 
-Feel free to also attach a package report using [DevMiner's package-size-calculator](https://github.com/TheDevMinerTV/package-size-calculator) to the issue, to give an overview over the package's size and the changes you are proposing.
+Feel free to also attach a package report using [DevMiner's package-size-calculator](https://github.com/TheDevMinerTV/package-size-calculator) to the issue, to give an overview of the package's size and the changes you are proposing.
 
 ### Fixes
 
-We've now identified some dependency which introduces performance problems.
-
-Depending on the type of improvement we've identified, there are a few different approaches for providing a fix/change.
+We've now identified a dependency which introduces performance problems. Depending on the type of improvement we've identified, there are a few different approaches for providing a fix/change.
 
 #### Large subtrees
 
-For these dependencies, we should first find out if there's a reason we need them and not an alternative.
+For these dependencies, we should first find out if there's a reason the dependency is necessary and shouldn't be replaced with an alternative.
 
 Common things to check:
 
-- node `engines` specifying a node version low enough that this dependency supports (and alternatives don't).
-- there is no known alternative (good opportunity to make one!).
-- every deep dependency in the subtree is necessary (in many cases the tree may be deep but for good reason, to share modules, etc).
+- [`engines`](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#engines) specifying a lower Node version that this dependency supports, but alternatives don't
```

Comment:
should this be `than this dependency supports`?

Reply:
No, but I realize the overall wording is confusing. I can take another crack at it.

Reply:
it is 😅 

i forget what point it was trying to get across

we check `engines` for two reasons usually:

- if it is high enough that some dependencies are made redundant by native functionality
- if it is low enough that we can't yet move to a more modern alternative

i guess this was saying you should check if the constraint is low enough that alternatives can't be used? (the 2nd point)

Reply:
I have rewritten this section to try and be clearer from the start.